src/components/file-dropper/file-list-item.tsx

```diff
@@ -84,13 +83,17 @@ export const FileListItem = ({
         {fileCategories && (
           <FileCategoryDropdown
             defaultValue={t('career.file-category').toString()}
-            onChange={(event) =>
-              onFileCategorySelect(index, event.target.value)
+            onChange={(selectedOption) =>
+              onFileCategorySelect(index, selectedOption)
             }
           >
-            <option disabled>{t('career.file-category')}</option>
+            <DropdownOption value={t('career.file-category')} disabled>
+              {t('career.file-category')}
+            </DropdownOption>
             {fileCategories.map((category) => (
-              <option key={category}>{category}</option>
+              <DropdownOption value={category} key={category}>
+                {category}
+              </DropdownOption>
```

Comment:
That looks straight forward. How did it feel? Is it a good replacement?

Reply:
I think the changes definitely make sense since we were able to fix all the problems we had with native select. 
The only thing that bothers me is that on mobile you no longer have the select popout that was shown with the normal select.

Reply:
maybe there is some issue/workaround for reach ui because this is a pretty standard behaviour.